security-analysis/security-analysis-default/src/test/java/com/powsybl/security/impl/mocks/LoadFlowProviderMock.java~HEAD

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.impl.mocks;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.loadflow.LoadFlowProvider;
+import com.powsybl.loadflow.LoadFlowResult;
+import com.powsybl.loadflow.LoadFlowResultImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@AutoService(LoadFlowProvider.class)
+public class LoadFlowProviderMock implements LoadFlowProvider {
```

Comment:
@EtienneLt Be careful, this file is called LoadFlowProviderMock.java-HEAD ! This is what is causing the fail in compilation